packages/components/src/components/lineage/models.ts

```diff
@@ -178,16 +178,17 @@ export class LineageNode extends Record ({
 
     static create(lsid, values?: { [key:string]: any }): LineageNode {
         return new LineageNode({
-            children: LineageLink.createList(values.children),
-            cpasType: values.cpasType,
+            children: values ? LineageLink.createList(values.children) : undefined,
```

Comment:
If values is undefined, maybe just create the LineageNode with the lsid instead of repeating the ternary conditional for each field?